### ISSUE TITLE
[Vulkan] An error occurred initializing the video core

### ISSUE THREAD

**Issue by jpvanoosten** [yuzu-emu/yuzu#4107]
I am using edb2114 on Windows 10 (v 1909 build 18363.900), Titan RTX with the latest drivers from NVIDIA (446.14) and I'm getting an error dialog box when trying to launch a ROM with Vulkan enabled "An error occurred initializing the video core.".

I've tried running other Vulkan applications (the samples from the LunarG SDK for example) and those run fine so I suspect it is an issue with the initialization code in Yuzu.

I've attached the log file which simply states:

```
[   4.306611] Render.Vulkan <Error> video_core\renderer_vulkan\vk_device.cpp:IsSuitable:467: Device lacks a graphics and present queue
[   4.306617] Render.Vulkan <Error> video_core\renderer_vulkan\vk_device.cpp:IsSuitable:515: TITAN RTX is not suitable
[   4.306866] Core <Critical> core\core.cpp:Load:199: Failed to initialize system (Error 5)!
```

Thanks.

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/4799423/yuzu_log.txt)

**Comment by Morph1984** [comment]
Can you try #4140 and see if this fixes your issue?

**Comment by Morph1984** [comment]
I just realized that with graphics debugging enabled under the debug tab, it fails to init, #4140 fixes this.
Run yuzu without the enabling graphics debugging option.

**Comment by german77** [comment]
Fixed by #4140

**Comment by kavinkishore923** [comment]
Hi i am also having problems with yuzu whenever i open pokemon sword and shield there is an error which says 'An error occured initializing the video core" i have "Intel(R) Core(TM) i3-1005G1 CPU @ 1.20GHz   1.19 GHz 4.00 GB (3.79 GB usable) can you please tell me how to fix this error?
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/6806955/yuzu_log.txt)
